tools/orca/src/error.rs

```diff
@@ -5,7 +5,8 @@ pub enum Error {
     KanidmClient,
     ProfileBuilder,
     Tokio,
-    Interupt,
+    Interrupt,
     Crossbeam,
     InvalidState,
+    InvalidInput(String),
```

Comment:
I want to avoid encoding strings into error messages, because then you just end up witnh InvalidInput and 13 different possible string errors. So use a generic version (InvalidInput) or make a new enum variant with the specific error encoded. Generally with orca, because the user is there at the terminal, a generic-variant is fine since the error!() line is what encodes the error info.

Reply:
Yeah actually I just realized I ended up not needing that at all ☠️☠️. So never mind I guess. 
But I'll keep it in mind for the future :)